packages/gcom4/package.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# Copyright 2024 ACCESS-NRI
+# Based on gcom/package.py by scottwales 2023
+# and https://github.com/coecms/access-esm-build-gadi/blob/master/Makefile
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+import llnl.util.filesystem as fs
+
+
+class Gcom4(Package):
+    """
+    GCOM is a wrapper around multiprocessing libraries such as MPI
+    Gcom4 is a package for older versions of GCOM as used by UM vn7.3 in ACCESS ESM 1.5
+    """
+
+    homepage = "https://code.metoffice.gov.uk/trac/gcom"
+    git = "git@github.com:ACCESS-NRI/GCOM.git"
+
+    maintainers = ["penguian"]
```

Comment:
Spack is changing this to a function. See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers

Reply:
Will do.

Reply:
Done.